front/app/containers/Admin/projects/components/PDFExportModal/index.tsx

```diff
@@ -94,10 +120,15 @@ const PDFExportModal = ({ open, formType, onClose, phaseId }: Props) => {
     setLoading(true);
 
     try {
+      if (htmlPdfsActive) {
+        await updateCustomForm({
+          printStartMultiloc: formValues.print_start_multiloc,
+          printEndMultiloc: formValues.print_end_multiloc,
+        });
+      }
       await onExport(formValues);
       setLoading(false);
       onClose();
-      methods.reset();
```

Comment:
Redundant. This would reset the form on downloading the PDF, but that will get overwritten again when we set the form values with the new effect (on the right).